src/ape/managers/chain.py

```diff
@@ -618,16 +618,27 @@ def get(
 
         return contract_type
 
+    def get_container(self, contract_type: ContractType) -> ContractContainer:
```

Comment:
Is this necessary?

Reply:
Yes, it pretty much is. It has to do with importing `ContractContainer`.

We have always had this method, it was in the `BaseModel` before but I think makes more sense here next to other contract-class factory methods.

The reason we need this is because there are internal parts of the code base that need to create `ContractContainer` objects. We are unable to import `ContractContainer` because of cyclical imports. Thus, (many months ago during the Pydantic refactor), we added methods to the `BaseModel` `create_contract()` and `create_contract_container()`. Recently, you pointed out that the names of these methods are misleading and I agree so that is why I created this PR (since it is a breaking change in prep for 0.4).

So either we have to do local imports of `ContractContainer` whenever that is needed or we can have this method which makes things easier.

Reply:
Ah okay, makes sense